packages/kotti-ui/rollup.config.js

```diff
@@ -35,6 +35,7 @@ const plugins = ({ enableVisualizer, module }) => [
 		needMap: false,
 	}),
 	typescript({
+		abortOnError: false,
```

Comment:
I added this so I could see all compilation error at once.  If omitted, the build will stop at the first error thrown